dev/io.openliberty.transport.ssl_fat/fat/src/io/openliberty/transport/ssl/SSLOptionsTest.java

```diff
@@ -0,0 +1,341 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2023 IBM Corporation and others.
```

Comment:
I won't comment on every one but if there are any significant changes to any of the tests we should update the range to include 2025